### PR TITLE
Add skip secure option example in app access docs

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/5.0/application-access.md [hunk 1]
```diff
@@ -185,6 +185,16 @@ app_service:
      uri: "https://localhost:8001"
      public_addr: "jira.example.com"
 ```
+### Skip TLS certification verification
+
+  Skip validating TLS certificate for applications.  Useful for applications that are using self-signed certificates.
```
Comment (benarent) [inline docs/5.0/application-access.md:190]: I'v expanded the note to be more explicit in why/what this does. 

```suggestion
  Corporation often use self-signed certificates for internal applications. Teleport attempts to check if the certificates are signed by a CA (Certificate Authority). If using self-signed certificates use  `insecure_skip_verify: true` to skip this verification step. 
```

**Review by benarent — CHANGES_REQUESTED** [review]
Added more clarity via my suggestion.

**Review by webvictim — APPROVED** [review]
LGTM pending my/Ben's changes

File: docs/5.0/application-access.md [hunk 2]
```diff
@@ -185,6 +185,16 @@ app_service:
      uri: "https://localhost:8001"
      public_addr: "jira.example.com"
 ```
+### Skip TLS certificate verification
+
+  Corporations often use self-signed certificates for internal applications. Teleport attempts to check if the certificates are signed by a trusted CA (Certificate Authority). If using self-signed certificates use  `insecure_skip_verify: true` to skip this verification step.
```
Comment (webvictim) [inline docs/5.0/application-access.md:190]:
```suggestion
  Corporations often use self-signed certificates for internal applications. Teleport attempts to check if the certificates are signed by a trusted CA (Certificate Authority). If using self-signed certificates use `insecure_skip_verify: true` to skip this verification step.
```

**Pull request merged**